docs/dev/protocols.md

```diff
@@ -149,47 +149,17 @@ The CQL protocol support can be disabled altogether by setting the
 
 These option names were chosen for backward-compatibility with Cassandra
 configuration files: they refer to CQL as the "native transport", to
-contrast with the older Thrift protocol (described below) which wasn't
-native to Cassandra.
+contrast with the older Thrift protocol which wasn't
+native to Cassandra. The thrift protocol was once supported by Scylla,
+but the support of this protocol was later deprecated and removed.
 
 There is also a `rpc_address` configuration option to set the IP address
 (and therefore network interface) on which Scylla should listen for the
 CQL protocol. This address defaults to `localhost`, but in any setup except
-a one-node test, should be overridden. Note that the same option `rpc_address`
-applies to both CQL and Thrift protocols.
+a one-node test, should be overridden.
 
 TODO: there is also `rpc_interface` option... Which wins? What's the default?
 
-## Thrift client protocol
-
-The Apache Thrift protocol was early Cassandra's client protocol, until
-it was superseded in Cassandra 1.2 with the binary CQL protocol. Thrift
-was still nominally supported by both Cassandra and Scylla for many years,
-but was recently dropped in Cassandra (version 4.0) and is likely to be
-dropped by Scylla in the future as well, so it is not recommended for new
-applications.
-
-By default, Scylla does not enable the Thrift server. In order to use it,
-it must be explicitly enabled by setting the `start_rpc` configuration option
-to true.
-
-When Thrift is enabled, by default scylla listens to the Thrift protocol on port 9160,
-which can be configured via the `rpc_port` configuration option. Again, this confusing
-name was used for backward-compatibility with Cassandra's configuration files.
-Cassandra used the term "rpc" because Apache Thrift is a remote procedure
-call (RPC) framework. In Scylla, this name is especially confusing, because
-as mentioned above, Scylla's internal communication protocol is based on
-Seastar's RPC, which has nothing to do with the "`rpc_port`" described here.
```

Comment:
Can you please find a way to preserve this paragraph which explains the confusing term "rpc" which it seems we still use in some places (like rpc_address)? It will continue to confuse generations of Scylla developers, and as long as it exists, we should explain it.

Reply:
i was under the impression that if we remove a feature, we should drop the whole document documenting it. but since the target audience of this document is developer, so i am preserving the paragraph explaining  `rpc` and also noting that thrift protocol is not supported anymore.

Reply:
I would love us to remove every reference to "rpc" which isn't the inter-node communication. But sadly, it seems we won't. We will still have "rpc_address" and possibly other stuff using this outdated term. So I just think we need to explain it somewhere in this document. I think it be moved (and edited appropriately) to the end of "CQL client protocol" section, right after the paragraph introducing "rpc_address". We should keep the explanation what this name "rpc" originally meant, to clarify that `rpc_address` is **not** about the "RPC" (internal communication) protocol. Confusingly, the internal RPC protocol listens on `listen_address`, while the CQL protocol listens on `rpc_address`. In a sane world, these names would be reversed.